app/views/static_pages/index.html.erb

```diff
@@ -37,6 +37,30 @@ $(function(){
         <%= button_to 'Join as black player', game_path(game), method: :patch, class: 'btn btn-primary join-button'%>
       <% end %>
     </div>
-    
+
+  <% end %>
+</div>
+
+<div class="clear"></div>
+
+<div class='game-index-div'>
+  <h1>Your Active Games</h1>
+
+  <%@user.games.each do |game| %>

```

Comment:
Instead of having to define @user. You should have `current_user` available to you in the view.

```
<% if user_signed_in? %>
  <% current_user.games.each %>
```